test/dlc_tests/container_tests/bin/dgl_tests/testMXNetDGL

```diff
@@ -4,7 +4,9 @@ HOME_DIR=/test
 BIN_DIR=${HOME_DIR}/bin
 LOG_DIR=${HOME_DIR}/logs
 
-git clone https://github.com/dmlc/dgl.git ${HOME_DIR}/artifacts/dgl
+DGL_RELEASE_TAG=$(python -c "import dgl; print(dgl.__version__)")
+
+git clone -b ${DGL_RELEASE_TAG} https://github.com/dmlc/dgl.git ${HOME_DIR}/artifacts/dgl
```

Comment:
The release branch usually is 0.N.x, e.g., 0.4.x or 0.3.x
You need to handle this.

Reply:
thanks, was hoping I could use the release tag but looks like i need to use the release branch